README.md

```diff
@@ -63,6 +63,41 @@ Note: x64 Ruby for Windows is known to have some compatibility issues.
 
 # Getting Started with Azure Resource Manager Usage (Preview)
 
+## Prerequisite
```

Comment:
I think would place this section after the "Install the rubygem packages" section, as that one provides context on authentication. Then we can talk about how to set this for usage in the packages.'